Source/WebInspectorUI/UserInterface/Views/SpreadsheetStyleProperty.js

```diff
@@ -475,6 +475,27 @@ WI.SpreadsheetStyleProperty = class SpreadsheetStyleProperty extends WI.Object
         this._cssDocumentationPopover = null;
     }
 
+    // InlineSwatch delegate
+
+    inlineSwatchColorVariableCompute (swatch, variableName = null)
```

Comment:
ditto above about renaming this to `inlineSwatchGetMatchingColorVariable`

style: unnecessary space between a function's name and the `(`